utils/githooks/pre-commit

```diff
@@ -1,21 +1,47 @@
 #!/bin/bash
 
-# A git hook to validate  and correct the copyright date in source files.
+# A git hook to validate and correct the copyright date in source files.
 if [ -n "${NO_PRE_COMMIT:=""}" ]; then
-    exit 0
+	exit 0
 fi
 
 regex='(^[[:blank:]]*[\*/]*.*)((Copyright[[:blank:]]*)([0-9]{4})(-([0-9]{4}))?)([[:blank:]]*(Intel.*$))'
 year=$(date +%Y)
 errors=0
+targets=(
+	# code source files
+	'*.c'
+	'*.h'
+	'*.go'
+	'*.py'
+	'*.proto'
+	'*.java'
+	'*.yml'
+	'*.yaml'
+	'*.sh'
+	'*.bash'
+	# DevOps files
+	'Makefile'
+	'Jenkinsfile'
+	'SConscript'
+	'Dockerfile*'
+	# Documentation files
+	'*.txt'
+	'*.md'
+	'README*'
+	'NOTICE*'
+	'LICENSE*'
+	'copyright'
+	# Miscellaneous files
+	'.env'
+)
 
-files=$(git diff --diff-filter=AM --name-only --cached -- \
-       '*.c' '*.h' '*.go' 'Makefile' '*.py' '*.proto' '*.java')
+files=$(git diff --diff-filter=AM --name-only --cached -- ${tagets[@]})
```

Comment:
(lint) Double quote array expansions to avoid re-splitting elements. [SC2068]
(lint) tagets is referenced but not assigned (did you mean 'targets'?). [SC2154]